src/providers/config-service.ts

```diff
@@ -17,6 +17,9 @@ export class ConfigService {
   URI_PROTOCOL = '/protocol.json'
   URI_QUESTIONNAIRETYPE = '_armt'
   URI_QUESTIONNAIREFORMAT = '.json'
+  aRMTDef_Prod = 'master'
+  arMTDef_Test = 'test'
+  useTempARMTRepo = true
```

Comment:
Can this be added in the `defaultConfig.ts` to easily control this for production and dev

Reply:
Sure let me move it.